src/PinAssignment/PortsLoader.cpp

```diff
@@ -33,21 +32,23 @@ PortsLoader::PortsLoader(PortsModel *model, QObject *parent)
 
 PortsLoader::~PortsLoader() {}
 
-bool PortsLoader::load(const QString &file) {
-  if (!m_model) return false;
+std::pair<bool, QString> PortsLoader::load(const QString &file) {
+  if (!m_model) return std::make_pair(false, "Ports model is missing");
```

Comment:
Same comment as before.

Reply:
This is something internal that never should happened. what description do you think might be good?